multiraft/transport.go

```diff
@@ -23,6 +23,8 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/cockroachdb/cockroach/gossip"
```

Comment:
multiraft shouldn't import gossip (it is at least in principle independent from the rest of cockroach). This cockroach-specific RPCTransport should live elsewhere, probably either cockroach/rpc or cockroach/server.